config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'products/index'
```

Comment:
Remove the auto-generated routes or send `--skip-routes` when you run `rails g controller`